Libraries/LibTLS/TLSv12.h

```diff
@@ -69,9 +69,11 @@ class TLSv12 final : public Core::Socket {
 
     static ErrorOr<NonnullOwnPtr<TLSv12>> connect_internal(NonnullOwnPtr<Core::TCPSocket>, ByteString const&, Options);
 
-    SSL_CTX* m_ssl_ctx { nullptr };
-    SSL* m_ssl { nullptr };
-    BIO* m_bio { nullptr };
+    void handle_fatal_error();
+
+    SSL_CTX* m_ssl_ctx;
+    SSL* m_ssl;
+    BIO* m_bio;
```

Comment:
Why did the default init values vanish? :eyes:

Reply:
CLion gets mad and thinks they are always null and grays out all branches...

Reply:
I'd rather have the default values tbh

Reply:
They are all initialized in the constructor so it shouldn't be a big problem, right? The code becomes quite unworkable in CLion:

![image](https://github.com/user-attachments/assets/76d55504-1619-4f08-88ae-c5f0172813b8)

We probably shouldn't adapt to workaround bugs in the IDE, but is there any downside in not having them?

Reply:
not _really_, but it's a policy I'd rather not break just because an IDE is being silly

Reply:
Added them back